.github/workflows/ci.yml

```
@@ -30,12 +34,155 @@ jobs:
+          mkdir -p dist/windows
+          
+          # Copy executables
+          cp obj/blackbox_decode.exe dist/windows/
+          cp obj/blackbox_render.exe dist/windows/
+          cp obj/encoder_testbed.exe dist/windows/
+          
+          # Function to copy DLL dependencies
+          copy_dll_deps() {
+            local exe_file="$1"
+            echo "Analyzing dependencies for $exe_file..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)